src/en/get-involved.md

```diff
@@ -64,28 +63,4 @@ Provide any of the following for each component or pattern:
 
 Here’s what we’ll soon be releasing. More on our <gcds-link href="{{ links.roadmap }}">roadmap</gcds-link>.
```

Comment:
With no items listed, it's odd to have it say "here's what we'll..." so maybe we can remove it?
```suggestion
More on our <gcds-link href="{{ links.roadmap }}">roadmap</gcds-link>.
```